client/apis/volumegroupsnapshot/v1alpha1/types.go

```diff
@@ -118,6 +118,11 @@ type VolumeGroupSnapshotStatus struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:resource:scope=Namespaced,shortName=vgs
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="ReadyToUse",type=boolean,JSONPath=`.status.readyToUse`,description="Indicates if all the individual snapshots in the group are ready to be used to restore a volume."
```

Comment:
`ready to be used to restore a volume` -> `ready to be used to restore a group of volumes`

Reply:
thanks, fix it